src/Sankey/SankeyLabel/SankeyLabel.tsx

```diff
@@ -72,9 +81,15 @@ export interface SankeyLabelProps {
    * Custom formatting for the label.
    */
   format?: (value: SankeyLabelFormatProps) => any;
+
+  /**
+   * 
+   */
+  ellipsis?: number | 'none' | 'auto'
```

Comment:
Missing comments. Whats the difference between `undefined` and `none`?

Reply:
Added the comment. When the value is `none`, then the text is not ellipsized no matter how long the width and whether it flows out of the container (it's an explicit option for use cases where showing the complete label might be important), while `undefined` would default to the default prop value `ellipsis: 'auto'`.